wallet/wallet.c

```diff
@@ -2277,3 +2278,77 @@ struct bitcoin_txid *wallet_transactions_by_height(const tal_t *ctx,
 	return txids;
 }
 
+void wallet_channeltxs_add(struct wallet *w, struct channel *chan,
+			   const int type, const struct bitcoin_txid *txid,
+			   const u32 input_num, const u32 blockheight)
+{
+	sqlite3_stmt *stmt;
+	stmt = db_prepare(w->db, "INSERT INTO channeltxs ("
+			  " channel_id,"
+			  " type,"
+			  " transaction_id,"
+			  " input_num"
+			  ", blockheight"
```

Comment:
Inconsistent comma usage

Reply:
Another wrinkle of comma placement, is that one notices that SQL consistently, uses longer command keywords, than the keywords for clauses: `SELECT` vs `WHERE` vs `ORDER` vs `FROM`; `INSERT` vs `VALUES`; `UPDATE` vs `WHERE` vs `SET`, etc.

    SELECT col1
         , col2
      FROM table
     WHERE col3 = 'whatever'
       AND col4 = 42
         ;

This has the advantage of never requiring spaces before the end of strings being concatenated, since everything after the initial command keyword gets an extra space before it:

    "SELECT col1"
    "     , col2"
    "  FROM table"
    " WHERE col3 = 'whatever'"
    "   AND col4 = 42"
    "     ;"

Reply:
That's a bit much imho, with the spacing that I added we actually get the same spacing as if we'd written it in the shell (`.schema` will show excessive spaces in the `CREATE` statement for example, which is hard to read).